typings/index.d.ts

```diff
@@ -1,27 +1,39 @@
 // Definitions by: Wu Haotian <https://github.com/whtsky>
 export interface IUserOptions {
-    allAtOnce?: boolean;
-    delay?: number;
+  allAtOnce?: boolean;
+  delay?: number;
 }
 
 export interface ITabUserOptions {
-    shift?: boolean;
-    focusTrap?: Document | Element;
+  shift?: boolean;
+  focusTrap?: Document | Element;
 }
 
 export type TargetElement = Element | Window;
 
+export type FilesArgument = File | File[];
+
+export type UploadInitArgument = {
+  clickInit?: MouseEventInit;
+  changeInit?: Event;
```

Comment:
I'm not sure in this type, took it from [there](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/testing-library__dom/events.d.ts#L86)